app/src/lib/git/rev-list.ts

```diff
@@ -4,14 +4,29 @@ import { Repository } from '../../models/repository'
 import { Branch, BranchType, IAheadBehind } from '../../models/branch'
 
 /**
- * Convert two refs into the Git range syntax.
+ * Convert two refs into the Git range syntax representing the set of commits
+ * that are reachable from `to` but excluding those that are reachable from
+ * `from`.
  *
  * Each parameter can be the commit SHA or a ref name.
  *
  * @param from The start of the range
  * @param to The end of the range
  */
-export function asRange(from: string, to: string) {
+export function asDistinctRange(from: string, to: string) {
+  return `${from}..${to}`
```

Comment:
You may note here that `asRange` was previously a wrapper for `...` - after reading the docs I think these usages should be `..` because we want the unique commits between `from` and `to`.

I've dropped the `ready-for-review` label here to ponder and test some more to ensure this is the right functionality for the compare tab experience, let me know if you have any other feels.

Reply:
Nevermind, the existing behaviour was accurate - I'll update this PR to make it clearer.